addons/point_of_sale/static/src/js/db.js

```diff
@@ -385,7 +387,9 @@ var PosDB = core.Class.extend({
             var r = re.exec(this.category_search_string[category_id]);
             if(r){
                 var id = Number(r[1]);
-                results.push(this.get_product_by_id(id));
+                const product = this.get_product_by_id(id);
+                if (!product.active) continue;
```

Comment:
shouldn't we also check that the product is available in POS?

Reply:
That's true.